package.json

```diff
@@ -99,6 +99,7 @@
     "jest-environment-node": "^26.6.2",
     "listr": "^0.14.3",
     "magic-string": "^0.25.7",
+    "merge-source-map": "^1.1.0",
```

Comment:
I have a question out to see if this needs to be added to https://github.com/ionic-team/stencil/blob/925d4e924264df424c3519f4c0a91b22356a2ea6/scripts/license.ts#L9 or not

Reply:
NVM - got confirmation we do (need to add it) - https://github.com/ionic-team/stencil/pull/3016

Reply:
Done https://github.com/ionic-team/stencil/pull/3005/files#diff-a559fe1aaf8746e145121786b79b8b86b1f87b4da31f52039453bfb12b8370f8R21